bigbluebutton-tests/playwright/whiteboard/draw.js

```diff
@@ -16,6 +16,11 @@ class Draw extends Page {
     const shapes1 = await this.getOuterHtmlDrawn();
 
     const wbBox = await this.getElementBoundingBox(e.whiteboard);
+
+    await this.page.mouse.click(wbBox.x + 0.3 * wbBox.width, wbBox.y + 0.3 * wbBox.height,{button: 'right'});
```

Comment:
let's perform the right-click a bit away from where we start drawing the shape, just to avoid any further bug
```suggestion
    await this.page.mouse.click(wbBox.x + 0.4 * wbBox.width, wbBox.y + 0.4 * wbBox.height, { button: 'right' });
```